### PR TITLE
cinder: Make tgtd listen on admin IP address only (bsc#828214)

### PR DESCRIPTION
**Pull request by vuntz**
http://bugzilla.suse.com/show_bug.cgi?id=828214

### CODE REVIEW

File: chef/cookbooks/cinder/templates/default/tgt.sysconfig.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Path:        Network/File systems/iSCSI
+## Description: tgt iSCSI server options
+## Type:        string
+## Default:     ""
+## ServiceRestart: tgtd
+#
+# Options for tgtd
+# set listen addr with --iscsi portal=192.0.2.1:3260
+#
+TGTD_OPTS="--iscsi portal=<%= @admin_ip %>"
```
Comment (tboerger) [inline chef/cookbooks/cinder/templates/default/tgt.sysconfig.erb]: What about the port? In the example above it's written like `--iscsi portal=192.0.2.1:3260`.

**Comment by tboerger** [comment]
One comment but LGTM

File: chef/cookbooks/cinder/templates/default/tgt.sysconfig.erb (hunk 1 above)
Comment (vuntz) [inline chef/cookbooks/cinder/templates/default/tgt.sysconfig.erb]: I think it's optional, and the CI seems to agree. I can add it if desired, though.

Comment (tboerger) [inline chef/cookbooks/cinder/templates/default/tgt.sysconfig.erb]: If you can confirm that it works fine I have no problem with it.

**Comment by rsalevsky** [comment]
+1

**Comment by mjura** [comment]
Ehhh, admin network is not right for exporting and using storage shares! In most cases users defined `admin` on really slow network interfaces like 100Mb/s or 1Gb/s. Why don't we want to use tgt in `storage` network? For me this is proper place for this.

**Comment by vuntz** [comment]
@mjura this is really unrelated to this PR. At the moment, `admin` is already what is being used by default as the nodes are not on the `storage` network...

File: chef/cookbooks/cinder/templates/default/tgt.sysconfig.erb (hunk 1 above)
Comment (vuntz) [inline chef/cookbooks/cinder/templates/default/tgt.sysconfig.erb]: I checked, it is optional.

**Comment by vuntz** [comment]
Anyone else up for a second +1?

**Comment by aplanas** [comment]
+1

**Comment by toabctl** [comment]
+1

**Pull request merged**